.cspell/people.txt

```diff
@@ -20,6 +21,9 @@ Rudi
 Sahaj
 Schreuder
 Smirnov
+Staber
+tein
```

Comment:
We can delete this by updating the bibliography slightly (see later comment)

Reply:
Done.